docs/api-docs/tax/tax-settings.mdx

```diff
@@ -26,7 +26,8 @@ To get tax settings, send a request to the [Get tax settings](/docs/rest-managem
         "show_inclusive_in_control_panel": true,
         "invoice_price_display_strategy": "ZONE"
       },
-      "fallback_strategy": "DISABLE"
+      "fallback_strategy": "DISABLE",
+      "should_subtract_store_tax": false
```

Comment:
If the documentation is to show the default value then it would be good to update this value to `true`
as per https://github.com/bigcommerce/api-specs/pull/1146#discussion_r1177330208

Reply:
Thanks, I'll leave the example, though I did add a callout for the field (that links to the api spec)